_posts/2019-04-07-sr2019-tlc-crowned-champion.md

```diff
@@ -0,0 +1,86 @@
+
+Some of the more notable designs were:
+
+- Team TLC with their consistent, successful catapult.
+- Team SWI with their simple and effective grab-and-go robot which climbed the volcano with a single token.
+- Team CRB with their suction cup and catapult combination which unfortunately had technical issues for much of the competition.
+- Team HAB with their Hawaiian themed robot, topped with a 3D paper pineapple.
+
+The Final
+---------
+
+Student Robotics 2019 concluded with an intense match between PSC, HAM, TLC, and HRS. Team HAM were quick off the mark pushing their tokens into the scoring zones. Team HRS soon followed, violently shoving their tokens across the arena. Team PSC's robot slowly moved 4 of their tokens into a single scoring zone before unfortunately flipping onto its back. Meanwhile, Team TLC quietly moved towards the edge of the volcano and launched a token directly into the Caldera.
+
+<!-- Video of Final -->
+<iframe
+  class="center video"
+  src="https://www.youtube.com/embed/gOeP96OngE0"
+  frameborder="0"
+  width="100%"
+  allowfullscreen
+  >
+</iframe>
+
+We saw some brilliant robot designs this year, great work by all teams. We hope you enjoyed the competition as much as we did!
+
+If you didn't take part, or you want to enter again next year, the [sign up page](/schools/how_to_enter) for next year's competition will be up later in the year. Get a team together and start talking to your teachers now!
+
+Results
+-------
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1st Place             | [The Ladies' College](http://www.ladiescollege.com/)
+| 2nd Place             | [Hills Road Sixth Form College](http://www.hillsroad.ac.uk/)
+| 3rd Place             | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
+| Rookie Award          | [Eltham College](https://www.elthamcollege.london/)
+| Committee Award       | [South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
+| Robot and Team Image  | [Haberdashers' Aske's Boys' School](https://www.habsboys.org.uk/)
+| Online Presence       | [Collyer's](https://www.collyers.ac.uk/) ([Twitter](https://twitter.com/CollyersRobots))
+
+For full details on the awards, please see the [rulebook](/docs/resources/2019/rulebook.pdf).
+
+Of course, this competition would not be possible without the support of [The Motorola Solutions Foundation](https://www.motorolasolutions.com/en_us/about/company-overview/corporate-responsibility/motorola-solutions-foundation.html), our sponsor for this year, who enabled us to provide a fun and educational opportunity for students from a range of engineering disciplines. 
+
+<!--Media
+-----
+
+Photos can be found on the Student Robotics 2019 Competition [Flickr group](http://www.flickr.com/groups/sr2019/).-->
+
+_The SR Team_
+
+
```

Comment:
It might be worth including a "Notes to editors" section here, like the one on the [SR2016 post](https://studentrobotics.org/news/2016-05-22-rgs-victorious-sr2016/). We could likely copy/paste that one, though some parts will need tweaking:
- end of the first paragraph ("more than 50 teams and over 400 students") 
- comment about the timing of the competition ("end of the Easter holiday")
- link to the flickr group (maybe remove if you're sharing the link above?)